android-components/components/support/test/src/main/java/mozilla/components/support/test/rule/Helpers.kt

```diff
@@ -37,11 +37,10 @@ private const val DEFAULT_DISPATCH_TIMEOUT_MS = 60_000L
  * @see Dispatchers.setMain
  * @see TestDispatcher
  */
-@OptIn(ExperimentalCoroutinesApi::class)
```

Comment:
We still need this annotation for the `runTest` below. Surprised that CI here hasn't failed on it.

Reply:
`runTest` is no longer experimental since 1.7.0.
If you get a local warning in the IDE, you might need to do a gradle sync.